packages/terra-section-header/src/SectionHeader.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Arrange from 'terra-arrange';
+import styles from './SectionHeader.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Text to be displayed on the SectionHeader.
+   */
+  title: PropTypes.node.isRequired,
+  /**
+   * Callback function triggered when the accordion icon is clicked.
+   */
+  onClick: PropTypes.func,
+  /**
+   * Whether the accordion icon should be displayed in its 'open' or 'closed' position.
+   */
+  isOpen: PropTypes.bool,
+};
+
+const defaultProps = {
+  onClick: undefined,
+  isOpen: false,
+};
+
+const SectionHeader = ({
+  title,
+  onClick,
+  isOpen,
+  ...customProps
+}) => {
+  if ((process.env.NODE_ENV !== 'production') && (!onClick && isOpen)) {
+    // eslint-disable-next-line no-console
+    console.warn('\'isOpen\' are intended to be used only when \'onClick\' is provided.');
+  }
+
+  const attributes = Object.assign({}, customProps);
+
+  if (onClick) {
+    // makes the div tabable
```

Comment:
Personally, I wouldn't worry about adding comments that talk about basic html behavior.

Reply:
Done 9ecd844b93e8f694fa2cd309c4e345354148a47d